recipes/libgit2/all/conandata.yml

```diff
@@ -2,3 +2,6 @@ sources:
   "1.0.1":
     url: "https://github.com/libgit2/libgit2/archive/v1.0.1.tar.gz"
     sha256: "1775427a6098f441ddbaa5bd4e9b8a043c7401e450ed761e69a415530fea81d2"
+  "1.1.0":
+    url: "https://github.com/libgit2/libgit2/archive/v1.1.0.tar.gz"
+    sha256: "41a6d5d740fd608674c7db8685685f45535323e73e784062cf000a633d420d1e"
```

Comment:
Cosmetic review: Can you keep the same order (decending) than in `config.yml` please?